e2e/manual-tests/catalogue/catalogue-test_9.md

```diff
@@ -53,12 +53,12 @@ A naïve visitor to the data catalogue can click around in the catalogue and und
 |||test collection event 1B, description for test collection event 1B, 286, 1992-2000 --> | | |
 | 25 |DATASETS|List of datasets for this resource | | |
 |||Table with the following columns: Name, Description | | |
-|||test dataset for testCohort1, label for test dataset 1  --> | | |
-|||name of test dataset 2 for test cohort 1, label for test dataset 2 for test cohort 1 --> | | |
+|||test dataset for testCohort1,   test description for dataset 1  --> | | |
+|||name of test dataset 2 for test cohort 1,  description for test dataset 2 for test cohort 1 --> | | |
```

Comment:
Comment on the catalogue and not your test, but should really read "List the networks in this cohort is involved"